src/main/java/org/mockito/internal/configuration/plugins/DefaultMockitoPlugins.java

```diff
@@ -53,7 +53,7 @@ public class DefaultMockitoPlugins implements MockitoPlugins {
                 MockitoLogger.class.getName(), "org.mockito.internal.util.ConsoleMockitoLogger");
         DEFAULT_PLUGINS.put(
                 MemberAccessor.class.getName(),
-                "org.mockito.internal.util.reflection.ReflectionMemberAccessor");
+                "org.mockito.internal.util.reflection.ModuleMemberAccessor");
```

Comment:
Since we flip the default here, I suppose we have to update the `mockito-subclass` artifact to set the `MemberAccessor` to reflection as well.

Reply:
> Since we flip the default here, I suppose we have to update the `mockito-subclass` artifact to set the `MemberAccessor` to reflection as well.

The `mockito-subclass`  seems to define member accessor [1] but `mockito-android` and `mockito-proxy` indeed relied on reflection, changed that, thanks!

[1] https://github.com/mockito/mockito/blob/main/subprojects/subclass/src/main/resources/mockito-extensions/org.mockito.plugins.MemberAccessor